test/Microsoft.IdentityModel.AotCompatibility.Tests/Microsoft.IdentityModel.AotCompatibility.Tests.csproj

```diff
@@ -4,8 +4,8 @@
 
   <PropertyGroup>
     <!-- This test only needs to run on .NET -->
-    <TargetFramework>net8.0</TargetFramework>
-    <TargetFrameworks Condition="'$(TargetNet9)'== 'True'">$(TargetFramework); net9.0</TargetFrameworks>
+    <TargetFrameworks>net8.0</TargetFrameworks>
+    <TargetFrameworks Condition="'$(TargetNet9)'== 'True'">$(TargetFrameworks);net9.0</TargetFrameworks>
```

Comment:
If `TargetNet9` is undefined we ended up with `TargetFrameworks` set from the common properties which is not what the test expects.